docs/rules/prefer-prototype-methods.md

```diff
@@ -0,0 +1,49 @@
+# Prefer use methods from the prototype instead of methods from an instance.
```

Comment:
This rule is an extension of the recommended eslint core rule [no-prototype-builtins](https://eslint.org/docs/rules/no-prototype-builtins) right? It would be good to link to that as a related rule in the rule doc.

Reply:
I don't think they are related.